hubspot/tasks.py

```diff
@@ -87,9 +86,13 @@ def check_hubspot_api_errors():
                 obj_id is not None
                 and "LINE_ITEM" in obj_type
                 and error_type == "INVALID_ASSOCIATION_PROPERTY"
-                and retry_invalid_line_association(obj_id, error_timestamp, details)
             ):
-                continue
+                try:
+                    line = Line.objects.get(id=obj_id)
+                except ObjectDoesNotExist:
+                    pass
+                else:
+                    HubspotLineResync.objects.get_or_create(line=line)
```

Comment:
Add `continue` in this `else` block so this particular type of error doesn't get logged.